utils/log_parser.py

```diff
@@ -217,7 +304,8 @@ def process(logLines, dumpPhases):
                 g = m.groups()
                 scopeName, kindName, nodeName = [
                     x.replace(" ", "") for x in g[:3]]
-                assert (kindName, nodeName) in NODES_MAP
+                assert (
+                    kindName, nodeName) in NODES_MAP, "The node that changes its operands, must already exist in nodes graph."
```

Comment:
Nits:
- delete the `,`
- `nodes graph` -> `node graph`?

Also, what does this assert message mean?

Reply:
i change the assertion to be "This node must already exist in node graph."